### PR TITLE
repository package

### PR DESCRIPTION
**Pull request by henderiw**
this is the PR to create a repository for use within nephio.
This package is used for provisioning any repo we use in Nephio: mgmt repo, Mgmt-staging and workload repos

### CODE REVIEW

**Comment by henderiw** [comment]
The only open point here is we probably need to agree on the GIT URL name.

File: repository/Kptfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: nephio-repo
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: Provisions a Gitea repo for use within Nephio
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/starlark:v0.4.3
+    configPath: set-values.yaml
```
Comment (johnbelamaric) [inline repository/Kptfile]: missing newline

File: repository/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# cluster
```
Comment (johnbelamaric) [inline repository/README.md]: should be "repository"

File: repository/Kptfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: nephio-repo
```
Comment (johnbelamaric) [inline repository/Kptfile:4]: should match the dir name

File: repository/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+
+## Description
+
+Provisions a Gitea repo for a Nephio cluster
+The repo will be named after the package name
```
Comment (johnbelamaric) [inline repository/README.md]: The repo is not named the same as the package, it uses the cluster name, right?

**Review by johnbelamaric — APPROVED** [review]
A few nits, but non-blocking.

/appove

File: repository/Kptfile (hunk 1 above)
Comment (henderiw) [reply repository/Kptfile]: fixed

File: repository/README.md (hunk 2 above)
Comment (henderiw) [reply repository/README.md]: fixed

**Pull request merged**